alpha/declcfg/load_test.go

```diff
@@ -337,6 +337,182 @@ func TestLoadFS(t *testing.T) {
+					},
+					{
+						Schema:  "olm.bundle",
+						Name:    "etcdoperator.v0.9.2-clusterwide",
+						Package: "etcd",
+						Image:   "quay.io/operatorhubio/etcd:v0.9.2-clusterwide",
+						Properties: []property.Property{
+							{Type: "olm.package", Value: json.RawMessage(`{"packageName":"etcd","version":"0.9.2-clusterwide"}`)},
+							{Type: "olm.gvk", Value: json.RawMessage(`{"group":"etcd.database.coreos.com","kind":"EtcdBackup","version":"v1beta2"}`)},
+							{Type: "olm.skipRange", Value: json.RawMessage(`">=0.9.0 <=0.9.1"`)},
+							{Type: "olm.skips", Value: json.RawMessage(`"etcdoperator.v0.6.1"`)},
+							{Type: "olm.skips", Value: json.RawMessage(`"etcdoperator.v0.9.0"`)},
+							{Type: "olm.channel", Value: json.RawMessage(`{"name":"clusterwide-alpha","replaces":"etcdoperator.v0.9.0"}`)},
+						},
+						RelatedImages: []RelatedImage{{Name: "etcdv0.9.2", Image: "quay.io/coreos/etcd-operator@sha256:c0301e4686c3ed4206e370b42de5a3bd2229b9fb4906cf85f3f30650424abec2"}},
+					},
+					{
+						Schema:  "olm.bundle",
+						Name:    "etcdoperator.v0.9.4",
+						Package: "etcd",
+						Image:   "quay.io/operatorhubio/etcd:v0.9.4",
+						Properties: []property.Property{
+							{Type: "olm.package", Value: json.RawMessage(`{"packageName":"etcd","version":"0.9.4"}`)},
+							{Type: "olm.package.required", Value: json.RawMessage(`{"packageName":"test","versionRange":">=1.2.3 <2.0.0-0"}`)},
+							{Type: "olm.gvk", Value: json.RawMessage(`{"group":"etcd.database.coreos.com","kind":"EtcdBackup","version":"v1beta2"}`)},
+							{Type: "olm.gvk.required", Value: json.RawMessage(`{"group":"testapi.coreos.com","kind":"Testapi","version":"v1"}`)},
+							{Type: "olm.channel", Value: json.RawMessage(`{"name":"singlenamespace-alpha","replaces":"etcdoperator.v0.9.2"}`)},
+						},
+						RelatedImages: []RelatedImage{{Name: "etcdv0.9.2", Image: "quay.io/coreos/etcd-operator@sha256:66a37fd61a06a43969854ee6d3e21087a98b93838e284a6086b13917f96b0d9b"}},
+					},
+					{
+						Schema:  "olm.bundle",
+						Name:    "etcdoperator.v0.9.4-clusterwide",
+						Package: "etcd",
+						Image:   "quay.io/operatorhubio/etcd:v0.9.4-clusterwide",
+						Properties: []property.Property{
+							{Type: "olm.package", Value: json.RawMessage(`{"packageName":"etcd","version":"0.9.4-clusterwide"}`)},
+							{Type: "olm.gvk", Value: json.RawMessage(`{"group":"etcd.database.coreos.com","kind":"EtcdBackup","version":"v1beta2"}`)},
+							{Type: "olm.channel", Value: json.RawMessage(`{"name":"clusterwide-alpha","replaces":"etcdoperator.v0.9.2-clusterwide"}`)},
+						},
+						RelatedImages: []RelatedImage{{Name: "etcdv0.9.2", Image: "quay.io/coreos/etcd-operator@sha256:66a37fd61a06a43969854ee6d3e21087a98b93838e284a6086b13917f96b0d9b"}},
+					},
+					{
+						Schema: "olm.bundle",
+					},
+				},
+				Deprecations: map[string]Deprecation{
```

Comment:
Why does this test load anything other than deprecations?

Reply:
Right now it's not so significant, but we will build on these to validate other characteristics where we validate the deprecations against the other declcfg content (for e.g. to ensure that deprecations only refer to objects present in the catalog).

Reply:
Perhaps scope this to what is needed today?

Reply:
Eh, my goal with this "stuff"+deprecations test was just that: that adding the new schema did not impact existing parsing.